easybuild/easyconfigs/n/nvtop/nvtop-1.0.0-fosscuda-2018b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '1.0.0'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['https://github.com/Syllo/nvtop/archive/%(version)s.tar.gz']
```

Comment:
echo'ing @migueldiascosta remark again here: this should be

```python
sources = ['%(version)s.tar.gz']
```

It'll work as is, but it'll re-download every time this easyconfig is installed (I think).

Reply:
Ahh, my misunderstanding.  Fixed.